sdc/tests/test_sdc_numpy.py

```diff
@@ -359,6 +360,104 @@ def sdc_impl(a):
             with self.subTest(data=case):
                 np.testing.assert_array_equal(ref_impl(array0), sdc_func(array1))
 
+    def test_fillna_numeric_inplace_false(self):
```

Comment:
You could combine tests `test_fillna_numeric_inplace_false` and `test_fillna_numeric_inplace_true` by moving generic code to separate method something like that:
```python
def _test_fillna_numeric_inplace(self, pyfunc, cfunc):
    # generic code
```
Then call the method in the tail of both tests.

Reply:
@densmirn The assertion code block of these tests are not identical (inplace=True methods modify original arrays), so this new **_test_fillna_numeric** func has to accept inplace arg and alter behavior based on it's value (made it this way).